src/libs/data-utils.js

```diff
@@ -293,20 +294,22 @@ export const EntityUploader = class EntityUploader extends Component {
             onClick: () => this.doUpload()
           }, ['Upload'])
         }, [
-          'The first column header must be:',
-          div({ style: { fontFamily: 'monospace', margin: '0.5rem', fontWeight: '600' } }, ['entity:[type]_id']),
-          'where ',
-          span({ style: { fontFamily: 'monospace', fontWeight: '600' } }, ['[type]']),
-          ` is the desired name of the data table in ${getAppName()}. For example, to create or update a `,
-          span({ style: { fontFamily: 'monospace', fontWeight: '600' } }, ['participant']),
-          ' table use ',
-          span({ style: { fontFamily: 'monospace', fontWeight: '600' } }, ['entity:participant_id']),
-          div({ style: { margin: '0.5rem 0 1rem' } }, ['All of the values in the ID column must be unique.']),
+          div({
+            style: { padding: '0 0 1rem' }
+          }, [
+            'Choose the data import option below. For more info on the table',
+            h(Link, {
+              'aria-label': 'Click for table info',
```

Comment:
Since this is already a text link, it would be better for overall accessibility to tweak the language so the link could be more self-descriptive, instead of adding a separate aria label. Something like: 'Learn more about data tables.'